tokens/governance/proposals.md

```diff
@@ -32,51 +29,65 @@ Moonbeam的治理系统正在更新中。下一个阶段的治理称为OpenGov (
 
 --8<-- 'text/governance/lead-in-definitions.md'
 
-请确保您查看每个网络和track的[治理参数](/learn/features/governance/#governance-parameters-v2)。
+请确保您查看每个网络和track的[治理参数](/learn/features/governance/#governance-parameters-v2){target=_blank}。
 
 ## 提案步骤 {: #roadmap-of-a-proposal }
 
 本教程将涵盖提案步骤图的前几个步骤，如下图突出显示的步骤所示。您将学习如何提交提案想法至[Moonbeam社区论坛](https://forum.moonbeam.foundation/){target=_blank}、提交原像以及使用原像哈希在链上提交提案。
 
-您可以在[治理概览页面的OpenGov (Governance v2)提案步骤](/learn/features/governance/#roadmap-of-a-proposal-v2){target=_blank}部分找到详细的解释。
+您可以在[治理概览页面的OpenGov提案步骤](/learn/features/governance/#roadmap-of-a-proposal-v2){target=_blank}部分找到详细的解释。
+
+![Proposal Roadmap](/images/tokens/governance/proposals/proposal-roadmap.png)
```

Comment:
```suggestion
![Proposal Roadmap](/images/tokens/governance/proposals/governance-proposal-roadmap.png)
```

Reply:
`governance-proposal-roadmap.png` was deleted in the corresponding english PR (https://github.com/moonbeam-foundation/moonbeam-docs/pull/718/files#diff-1f558198c6fb0c433ad2ad4ff1dd44937ef46e395b0dae5f44d20cb555d89738) and replaced with `proposal-roadmap.png`